### ISSUE TITLE
IMDB parsing

### ISSUE THREAD

**Issue by sguyennet** [hwkns/macguffin#10]
IMDB modified their pages and the scripts now display this error:

```
Traceback (most recent call last):
  File "./auto_upload.py", line 111, in <module>
    dry_run=args.dry_run,
  File "/home/tc/scripts/uploads/upload.py", line 93, in start
    self.get_metadata()
  File "/home/tc/scripts/uploads/upload.py", line 269, in get_metadata
    self.imdb.get_metadata()
  File "/home/tc/scripts/metadata/imdb.py", line 81, in get_metadata
    self.get_main_metadata()
  File "/home/tc/scripts/metadata/imdb.py", line 92, in get_main_metadata
    header_spans = overview_top.h1.find_all('span')
AttributeError: 'NoneType' object has no attribute 'h1'
```

**Comment by hwkns** [comment]
Should be fixed in 7c8ed7a -- can you verify that it works for you @sguyennet ?

Thanks for reporting!

**Comment by sguyennet** [comment]
Perfect!
Thanks.
